src/values/Sentence.js

```diff
@@ -35,4 +35,23 @@ Sentence.prototype.getLocale = function() {
 	return this._locale;
 };
 
+/**
+ * Returns the passiveness of a sentence.
+ *
+ * @returns {boolean} returns true if passive, otherwise returns false.
```

Comment:
You could leave out 'returns' at the beginning + please start with a capital letter

Reply:
Agreed, fixed.